rust/cymbal/src/symbol_store/saving.rs

```diff
@@ -93,6 +93,23 @@ impl<F> Saving<F> {
 
         self.s3_client.put(&self.bucket, &key, data).await?;
         record.save(&self.pool).await?;
+        // We just saved new data for this symbol set, which invalidates all our previous stack frame resolution results,
+        // so delete them
+        let deleted: u64 = sqlx::query_scalar!(
+            r#"WITH deleted AS (DELETE FROM posthog_errortrackingstackframe WHERE symbol_set_id = $1 RETURNING *) SELECT count(*) from deleted"#,
+            record.id // The call to save() above ensures that this id is correct
+        )
+        .fetch_one(&self.pool)
+        .await?.map_or(0, |v| {
+            v.max(0) as u64
+        });
```

Comment:
Could these frames be cached in cymbal? We should probably remove any cached frame when we delete it unless we handle that elsewhere